src/pkg/utils/io.go

```diff
@@ -183,6 +192,24 @@ func ReplaceTextTemplate(path string, mappings map[string]*TextTemplate, depreca
 			if template != nil {
 				value = template.Value
 
+				// Check if the value is a file type and load the value contents from the file
+				if template.Type == "file" {
+					if isText, err := IsTextFile(value); err != nil || !isText {
+						message.Warn("Refusing to load a non-text file for templating")
```

Comment:
Can we name the variable that we are skipping? Here and the other warnf below.

I feel like that would be useful information for the person seeing the warning message.